content/projects/bootcamp-writing-assignment/_index.md

```diff
@@ -29,56 +31,57 @@ If you are using ACode on your android device, you can press the "play" button i
 
 If you are running vscode you can right click on a markdown file and select "open preview" to see what your work looks like.
 
-### Why?
-
-There are a few reasons:
-
-1. We want to get to know you :)
-2. A lot of people dont really sit down and think about what they are trying to do in the long run. It's a really useful excercise
 
 ### The questions
 
-Please create a separate file for each of these. Remember to pay attention to your sentence structure, spelling and grammar.
+Please create a separate file for each of these. Remember to pay attention to your sentence structure, spelling and grammar. 
+
+If you need a little help with your grammar, [Grammarly](https://app.grammarly.com/) is a really useful service. Being able to communicate clearly and professionally in text is a super power.
 
 #### 1. Your history [200 - 500 words]
```

Comment:
Can we add something to this about time/financial commitments?

Reason being we need to know if they are freelancing/working/in uni full time or anything. Also we need to know if they can afford to work for a small stipend.

Reply:
I thought you said you would add the extra questions?